packages/eslint-plugin-office-addins/src/rules/no-context-sync-in-loop.ts

```diff
@@ -0,0 +1,30 @@
+import { TSESTree } from "@typescript-eslint/typescript-estree";
+
+export = {
+  name: "no-context-sync-in-loop",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      loopedSync: "Calling context.sync() inside a loop can lead to bad performance",
+    },
+    docs: {
+      description: "Calling context.sync() inside of a loop dramatically increates the time the code runs the more iterations that are run",
```

Comment:
I think the message can be changed to something more like:
"Each call of context.sync() in a loop dramatically increases the execution time of the code."

Reply:
Need a design/PM/Doc writer to give us specific text.

The performance impact of each individual call isn't really noticeable . . . and this rule and its documentation is specifically about loops.  The other rule in our list (calling context.sync()) would be we were we talk about the number of times it's called.